python/packages/nisar/workflows/dumpconfig.py

```diff
@@ -283,14 +283,32 @@ def init_argparse():
             "behavior for the source runconfig."
         ),
     )
-    
-    subparsers.add_parser(
+
+    gcov_sub = subparsers.add_parser(
         "gcov",
         parents=[geo_parse],
         formatter_class=help_formatter,
         help="Create a GCOV runconfig.",
     )
 
+    # This method enables adding an argument to an argument group in a parent parser of
+    # a subparser without adding it to that same group in other subparsers.
+    geocoding_group = next(
+        filter(
+            lambda group: group.title == geocoding_group_title,
+            gcov_sub._action_groups,
+        ),
+        None,
+    )
+    geocoding_group.add_argument(
+        "--rtc-dem-upsampling",
+        dest="rtc_dem_upsampling",
+        type=int,
+        help=(
+            "RTC DEM upsampling."
+        ),
```

Comment:
This looks pretty good! Just some small final documentation improvements:
```suggestion
        required=False,
        help=(
             "Sets the RTC DEM upsampling value. Setting this value overrides the "
             "automatic RTC setting behavior of the --nisar-defaults parameter."
        ),
```

And in the `--nisar-defaults` parameter help string:

```python
    geometry_group.add_argument(
        "--nisar-defaults",
        action="store_true",
        required=False,
        help=(
            "If given, all geospatial parameters (spacing and snap values) not "
            "provided by the user will be filled with default values in meters that "
            "are suited to the workflow and bandwidth of the frequencies in the RSLC "
            "product. Only valid for products with 5, 10, 20, or 77 MHz bandwidths for "
            "all frequencies. This is suggested for use in conjunction with the "
            "--generate-epsg parameter. For GCOV products, this argument may also "
            "modify the RTC value to the NISAR default value for their band."
        ),
    )
```

Reply:
Thank you, @Tyler-g-hudson ! These descriptions were updated in commits https://github.com/isce-framework/isce3/pull/120/commits/c6f5c21708703cb17bef01b10a99de49cb7af781 and https://github.com/isce-framework/isce3/pull/120/commits/102481bf6e8c54f1200f178f78cf02ec57671774.